.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: "\U0001F4DA Documentation"
-about: Report an issue with napari documentation
+about: Report an issue with napari code documentation
```

Comment:
I'd say not only *code* documentation, but all documentation should be reported there.

Reply:
But most of the documentation does not live there. Source files are in the docs repository.

Reply:
Oh sorry, I read that wrong - nevermind! 😄